coordinape/coordinape
# Feature/create nominee Hasura migration

Closes #455 

### Test Plan

Prerequisite for testing notifications : Telegram Bot + `chat_id` & discord webhook

1. Go to `http://localhost:3000/admin/circles` Settings page enable Vouching and set minimum vouches to 2
2. In the DB go to the circle you wish to test on and update `telegram_id` to your `chat_id` and the `discord_webhook` accordingly.
2. run this mutation in your hasura console.
```gql
mutation MyMutation {
  createNominee(payload: {address: "<address>", description: "<desc>", name: "<name>", circle_id: <circle_id>}) {
    nominee {
      id
      circle_id
      description
      name
      nominated_by_user_id
      nominated_date
      expiry_date
      ended
      created_at
      address
      vouches_required
    }
  }
}
```
3. You should recieve a notification in both discord and telegram chat to indicate that a user has been nominated.
4. go to `http://localhost:3000/vouching` and you should see the nomination appear on the page.
5. go back to `http://localhost:3000/admin/circles` Settings page and set the minimum vouches to 1.
6. Run the above mutation again with a different address.
7. This time you should 2 notifications each, first is the nomination message and the second is the message indicating the nomination has received enough vouches and user has been added to circle.
8. Go to `http://localhost:3000/admin/circles` and you should see the nomination added as a user in the table
